deploy/roles/deploy.builder/templates/service-assurance/elasticsearch/elasticsearch.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: "logging.openshift.io/v1"
+kind: "Elasticsearch"
+metadata:
+  name: "elasticsearch"
+spec:
+  managementState: "Managed"
+  nodeSpec:
+    image: {{ registry_path }}/{{ imagestream_namespace }}/ose-elasticsearch5:latest
+    resources:
+      limits:
+        cpu: 500m
+        memory: 1Gi
+      requests:
+        cpu: 500m
+        memory: 1Gi
+  nodes:
+  - nodeCount: 1
```

Comment:
This also needs to be configurable so that we don't have a limit of a single node with zero redundancy. I know a lot of other things are running in singular mode as well right now, but the ElasticSearch operator seems to have logic that can handle cluster setup.